Source/WebKit/UIProcess/XR/ios/WKARPresentationSession.mm

```diff
@@ -117,6 +122,7 @@ - (nullable instancetype)initWithSession:(ARSession *)session descriptor:(WKARPr
         _session = session;
         _sessionDescriptor = adoptNS([descriptor copy]);
         _device = adoptNS(MTLCreateSystemDefaultDevice());
+        _sessionEndRequested = false;
```

Comment:
if the property is to be made atomic, this would have to become self.sessionEndRequested